CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Changelog
 
 * [CHANGE] Update the gRPC dependency to v1.66.0 and deprecate the `grpc_server_recv_buffer_pools_enabled` option that is no longer supported by it. #580
+* [CHANGE] `ring.DoBatchWithOptions` (and `ring.DoBatch`) reports the cancellation cause when the context is cancelled instead of `context.Cancelled`.
```

Comment:
(To follow spelling used in Go)

```suggestion
* [CHANGE] `ring.DoBatchWithOptions` (and `ring.DoBatch`) reports the cancelation cause when the context is canceled instead of `context.Canceled`. #577 
```